web_m2x_options/README.rst

```diff
@@ -84,6 +92,6 @@ Your XML form view definition could contain::
 Note
 ----
 
-Double check that you have no inherited view that remote ``options`` you set on a field ! 
-If nothing work, add a debugger in the first ligne of ``get_search_result method`` and enable debug mode in OpenERP. When you write something in a many2one field, javascript debugger should pause. If not verify your installation.
+Double check that you have no inherited view that remove ``options`` you set on a field ! 
+If nothing works, add a debugger in the first line of ``get_search_result method`` and enable debug mode in OpenERP. When you write something in a many2one field, javascript debugger should pause. If not verify your installation.
```

Comment:
Please take the occasion to update the template to current OCA's: https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst